save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RequestUtils.kt

```diff
@@ -158,9 +157,11 @@ suspend fun WithRequestStatusContext.delete(
 @Suppress("EXTENSION_FUNCTION_WITH_CLASS")
 suspend fun WithRequestStatusContext.loadingHandler(request: suspend () -> Response) = run {
     setLoadingCounter { it + 1 }
-    val response = request()
-    setLoadingCounter { it - 1 }
-    response
+    val job = coroutineScope.async { request() }
```

Comment:
```suggestion
    val deferred = coroutineScope.async { request() }
```
Let's not call it `job` to avoid confusion with `launch`

Reply:
Yes, sure